cypress/e2e/setup-scripts/seed_collections_user.py

```diff
@@ -0,0 +1,52 @@
+"""DB seed script for e2e cypress collections tests."""
+
+import json
+import os.path
+
+from django.conf import settings
+from django.contrib.auth.models import User
+
+from apps.helpers import create_app_instance
+from apps.models import Apps
+from projects.models import Environment, Flavor, Project, ProjectTemplate
+from projects.tasks import create_resources_from_template
+
+cypress_path = os.path.join(settings.BASE_DIR, "cypress/fixtures")
+print(f"Now loading the json users file from fixtures path: {cypress_path}")  # /app/cypress/fixtures
+
+with open(os.path.join(cypress_path, "users.json"), "r") as f:
+    testdata = json.load(f)
+
+    userdata = testdata["collections_user"]
+
+    username = userdata["username"]
+    email = userdata["email"]
+    pwd = userdata["password"]
+
+    # Create a superuser because that's the one that can currently create collections
+    superuser = User.objects.create_superuser(username, email, pwd)
+    superuser.save()
+
+    # Create a project for apps to be included in the collection
+    project = Project.objects.create_project(
+        name="e2e-collections-test-proj", owner=superuser, description="e2e-collections-test-proj-desc"
+    )
+    project.save()
```

Comment:
I think it would be cool it we added a check if the user and project exists, and if not, we create it. I ran this script manually, then the tests crashed. Would make it a bit more robust i believe.

Reply:
Hm, the problem would be that if the project exists we don't know in what state - the previous test might have failed at different points in time (so it might already have some things that the test expects not to find etc.) so we'd need to remove the project and start over anyway.